packages/data-stores/src/support-queries/use-active-support-tickets-query.ts

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from 'react-query';
+import wpcomRequest from 'wpcom-proxy-request';
+import { SupportTicket } from './types';
+
+const ACTIVE_STATUSES = [ 'New', 'Open', 'Hold' ];
+
+export const useActiveSupportTicketsQuery = ( email: string, queryOptions = {} ) => {
```

Comment:
In general, when migrating code from Calypso, please try to use the moved code in Calypso and not duplicate it. I mean, this [hook](https://github.com/Automattic/wp-calypso/blob/trunk/client/data/help/use-active-support-tickets-query.js#L6). The duplicate in Calypso should be deleted and the one in the package should be the only copy.

If TypeScript gives you weird errors, it likely means there is a circular dependency (Calypso depends on help-center and help-center on Calypso). So if that happens, this hook need to be in data-stores and both Calypso and HC can use it.